test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/Dockerfile

```diff
@@ -2,16 +2,16 @@
 # This should be run from the root of the repo:
 # docker build --file test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/Dockerfile .
 
-ARG SDK_VERSION=6.0
+ARG SDK_VERSION=7.0
```

Comment:
This is updated as the project references asp.net core which is targeting .net7.0.